lib/web/oidcidp.go

```diff
@@ -32,6 +32,8 @@ import (
 const (
 	// OIDCJWKWURI is the relative path where the OIDC IdP JWKS is located
 	OIDCJWKWURI = "/.well-known/jwks-oidc"
+	// OktaJWKSURI is the relative path where the Okta JWKS is located
+	OktaJWKSURI = "/.well-known/jwks-okta"
```

Comment:
nit: spell out what the URI refers to
```suggestion
	OktaJWKSWellknownURI = "/.well-known/jwks-okta"
```